apps/web/src/components/ExperienceFormFields/WorkFields.tsx

```diff
@@ -78,14 +62,7 @@ const WorkFields = ({ labels }: SubExperienceFormProps) => {
             label={labels.team}
             name="team"
             type="text"
-            rules={{
-              maxLength: {
-                message: intl.formatMessage(errorMessages.overCharacterLimit, {
-                  value: 256,
-                }),
-                value: 255,
-              },
-            }}
+            rules={{ required: intl.formatMessage(errorMessages.required) }}
```

Comment:
Team is also optional

Per the changes made in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8830/files

Reply:
Oops 😅 Reverted in [revert making team required](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8894/commits/940c8b5979c2d49836b2221dbbd0c9e045660868)